scripts/ensmallen-release.sh

```diff
@@ -23,7 +23,8 @@ if [ "$#" -gt 5 ]; then
 fi
 
 # Make sure that the branch is clean.
-lines=`git diff | wc -l`;
+# Truncate leading whitespaces since wc -l on MacOS adds an extra \t.
```

Comment:
I think it's macOS but don't think people will be confused.

Reply:
It's `macOS`... No one will be confused. just don't name it OS X 😉

Reply:
Is it a roman numeral or not?  Do I pronounce it X or 10?  Question for the ages :smile: